samples/excel/42-range/range-copyfrom.yaml

```diff
@@ -110,12 +111,23 @@ script:
             });
         }
 
+        async function move() {
```

Comment:
Should this be a standalone sample? The name of this is copyfrom so I don't know that it'll be clear that move is included here. Maybe rename this to something like copyAndMove?

Reply:
I thought about separating it out, but I think it's better off here. It's part of the same "cut/copy/paste" workflow. I'll add the word "move" to the description to help with searches.